x-pack/plugin/esql/qa/testFixtures/src/main/resources/semantic_text.csv-spec

```diff
@@ -173,3 +173,946 @@ host:keyword | semantic_text_field:semantic_text
+3            | null
+;
+
+mvCount
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = mv_count(st_multi_value)
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:integer
+1            | 3
+2            | 2
+3            | null
+;
+
+mvDedupe
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = mv_dedupe(st_multi_value)
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:semantic_text
+1            | ["Hello there!", "This is a random value", "for testing purposes"]
+2            | ["nice to meet you", "bye bye!"]
+3            | null
+;
+
+mvFirst
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = mv_first(st_multi_value)
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:semantic_text
+1            | Hello there!
+2            | nice to meet you
+3            | null
+;
+
+mvLast
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = mv_last(st_multi_value)
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:semantic_text
+1            | for testing purposes
+2            | bye bye!
+3            | null
+;
+
+mvMax
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = mv_max(st_multi_value)
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:semantic_text
+1            | for testing purposes
+2            | nice to meet you
+3            | null
+;
+
+mvMin
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = mv_min(st_multi_value)
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:semantic_text
+1            | Hello there!
+2            | bye bye!
+3            | null
+;
+
+mvSlice
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = mv_slice(st_multi_value, 1, 2)
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:semantic_text
+1            | ["This is a random value", "for testing purposes"]
+2            | bye bye!
+3            | null
+;
+
+mvSort
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = mv_sort(st_multi_value, "ASC")
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:semantic_text
+1            | ["Hello there!", "This is a random value", "for testing purposes"]
+2            | ["bye bye!", "nice to meet you"]
+3            | null
+;
+
+mvZip
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = mv_zip(st_multi_value, st_multi_value, " + ")
+| KEEP _id, result
+| SORT _id
+;
+
+_id: keyword | result:keyword
+1            | ["Hello there! + Hello there!", "This is a random value + This is a random value", "for testing purposes + for testing purposes"]
+2            | ["nice to meet you + nice to meet you", "bye bye! + bye bye!"]
+3            | null
+;
+
+equalityWithConstant
```

Comment:
tests for [Binary operators](https://www.elastic.co/guide/en/elasticsearch/reference/master/esql-functions-operators.html#esql-binary-operators) - the ones that support string types